src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+phrase "the vanishing of the vanishing".
+
+> This can also be expressed thus: becoming is the vanishing of being into
+> nothing, and of nothing into being, and the vanishing of being and nothing in
+> general; but at the same time it rests on their being distinct. It therefore
+> contradicts itself in itself, because what it unites within itself is
+> self-opposed; but such a union destroys itself (Hegel 2010, 81/21.93-4).
+
+Hegel repeats the point, but sharpens the idea of an internal opposition within
+`becoming` itself. Its movement brings together contradictory impulses and is
+torn asunder by the unrelenting indeterminacy of `pure being` and `nothing`.
+
+`Becoming` is the attempt, one might say, to bring some coherence to the
+conflicting motions of the early categories by carefully noting what must be the
+case: `being` and `nothing` _are_ different, they _are_ the same and they vanish
+into one another. The tools by which `becoming` puts these divergent and
+incompatible ideas, such as moments, movement, coming-to-be and ceasing-to-be,
+are innovative and creative, but equally cannot contain the fury of negativity
+latent in within them. `Becoming` itself thus emerges and perishes.
+
+#### Not Nothing
+
+> This result is a vanishedness, but it is not _nothing_; as such, it would be
+> only a relapse into one of the already sublated determinations and not the
+> result of nothing _and of being_. It is the unity of being and nothing that
+> has become quiescent simplicity. But this quiescent simplicity is _being_, yet
+> no longer for itself but as determination of the whole (Hegel 2010, 81/21.94).
+
+The vanishing of the vanishing would return one to the category `nothing`, but
+Hegel claims that this is not so. The first reason is that this would only put
+the logic back a few steps and _those_ steps would lead precisely _back_ to the
+present circumstance. Now, it may be that the logic merely loops around
+endlessly between `nothing` and `becoming` from this point on, but even this
+loop would be a new result, and one that displays characteristics of stability
+not seen before. The second reason is that this relapse does not take into
+account the complete outcome of `being` and `nothing`, viz. how they are
+inseparable.
+
+It is exactly this inseparability that is left after the rise and fall of
+`becoming`: the vanishing of `being` and `nothing` cannot be coherently
+determined since it itself vanishes, but it remains a fact that these two pure
+categories are deeply entangled and the motion of their vanishing is matched
+only by the inertia of their belonging.
+
+`Being` and `nothing` are thus understood to be simply together, but this
+togetherness itself falls under the header of `being` since this unity _is_. As
+seen above with the specific movements, it is `being` that comes-to-be and
+ceases-to-be. Now a new being must be defined, since the being of `being` and
+`nothing` is no longer pure.
```

Comment:
I don't think I agree that this is why Hegel emphasises being over nothing. I think you're right to emphasise the is-ness of Being as an important factor. 
I also think Hegel is using a kind of shorthand by just writing Being because, as we have just seen, Being is indistinguishable from Nothing and so to talk just of Being looks we're ignoring that.

Reply:
Why do you think Hegel emphasises being over nothing then, if not for that reason? Basically, I see there's an intimation of existence coming to the fore here. 

Can you elaborate on the thing about the "shorthand"? I'm not sure I follow.

Reply:
I think your interpretation raises some interesting questions. 

The idea that the unity falls under header of "being" because this unity 'is' raises some questions. First of all, doesn't "nothing" also 'is' insofar as it is a determination of thought that is being investigated? It must have being (and, in fact, I think that's Hegel's point about the concept of Nothing), and so it must have being, in other words, it must 'is'. It is in this sense, that I think Hegel is using "being" as a shorthand for both "being" and "nothing". In fact, he's using it as a shorthand for determinate being. Just as when he distinguishes it as a "determination on its own" and a "determination of the whole". 

If the above is wrong, however, what does that tell us about the the determination of "nothing"? Why is it such a central determination in our understanding of "becoming" and the moments of coming-to-be and ceasing-to-be, but is irrelevant in the moment that follows from them? I'm not sure that quite makes sense.

All of the above, however, is nullified by what Hegel says in the next paragraph: 

"Becoming, as this transition into the unity of being and nothing, a unity which is in the form of being or has the form of the onesided immediate unity of these moments, is determinate being." (106, Miller). 

Hegel assuages all of the concerns I have raised above by saying that the form that it has is a onesided one - _i.e_ it ignores nothing. That is, at least, consistent. But, at this stage, I suppose my concern would be with the status of "nothing". What does Hegel mean by the form of being? Does he mean is-ness? Well, surely "nothing" has isness, otherwise, how do we think it? What exactly is meant by the form of being? Which by the way, sounds like a very determinate understanding of such an indeterminate determination.

Also, now that I am re-reading this section, I would query the following sentence: " it is `being` that comes-to-be and ceases-to-be". That is not quite right, it is nothing that comes to be and being that ceases to be.

Reply:
These are some really great reflections. It's made me make a new component called HotTopic which renders this: 

![image](https://github.com/user-attachments/assets/0765abdb-563c-441d-a853-9c4cb86e73f1)

I think we should move this to the discussion board, and come back to revise the article. What do you think?